### DISCUSSION TITLE
Best practice: Monica v4 vs v5

### DISCUSSION THREAD

**Discussion by OddSquirrel** [monicahq/monica#7321] [Q&A] [answered]
Hi there,

I recently discovered this wonderful project and wondered how to go about using it because it's all a bit confusing for this old noob.

First, there seems to be talk about Monica's successor, Chandler. Looking for Chandler leads me to an abandoned project, and it appears that Chandler transformed into Monica v5 instead.

It's also mentioned that Monica v4 won't receive any further updates, but it looks like it's actively being worked on.

While there are instructions for installing v4, I can't find all that much on v5.

So, in the end, I went and installed v4 in a Docker container, and it appears to work beautifully. But as I'm actively adding contacts and details, I wonder if I'll eventually be able to move these to v5 or if I'll have to redo the entire database, which would be a nightmare, of course.

It would be awesome if anybody could point me in the right direction, so I don't set up something now that I regret later.

**Comment by coolmule0** [comment, marked as answer]
[This thread](https://github.com/monicahq/monica/issues/6626) Has a lot of the details and information you are asking. Some of it is quite hidden away in comments and threads, so I will try to summarize from there your questions.

The Chandler repository was archived in favour of keeping the Monica repository. This allows the github stars and impact to be maintained. Now the main branch (v5) is Chandler and v4 is the old Monica.

There is a split of usage in Monica v4 and v5, so there is still activity in both areas.

The last comment of the repository author suggested they might not look at creating a v4 -> v5 importer in the near future (https://github.com/monicahq/monica/issues/6626#issuecomment-2002484416). The dev also recommends going straight to v5 https://github.com/monicahq/monica/discussions/6904#discussioncomment-7169080.

For instructions on v5 docker, there is an excellent `docker-compose` in this thread https://github.com/monicahq/monica/discussions/7153#discussion-6203825. There is also a single service `docker-compose` file in a discussion/issue, but I can't find it at the moment.

I have been using this project on and off, and recently got back into it with a v5 instance, so my information may not be accurate.

**Reply by OddSquirrel** [reply]
Thanks for taking the time to reply, I sincerely appreciate it. 👍

Looks like we're really in an awkward position. Both Monica and Monica-Next / Chandler seem to sputter away on life support, with no migration path on the horizon. That means possibly entering hundreds or thousands of people and relationships into an app that could sunset at any time, no matter which version you choose. And if you go for the stable Monica version, you'll lose everything because there's no migration. 😒

I mean, this isn't a Tic Tac Toe app, you build a treasure trove of information over years and years that can't just be recreated on another platform in a day.

Dang, I do wish I could help, but my programming skills are about on par with my Klingon fluency. 🙈
